kbchat/kbchat_test.go

```diff
@@ -103,82 +103,290 @@ func deleteWorkingDir(workingDir string) error {
 	return os.RemoveAll(workingDir)
 }
 
-var kbc *API
-var config Config
-
-func TestMain(m *testing.M) {
+func testSetup() (alice *API, config botConfig, dir string, oneOnOneChannel Channel, teamChannel Channel) {
```

Comment:
this setup and teardown only works for `alice`. she is hardcoded, which feels a little weird. maybe this method should take the name of the user to set up, even if alice is the only one we have right now. it's a really small tweak, and it'll make some of the later things read nicer (i.e. alice, ... := testSetup("alice")) or you could decouple the outside by having this be more like `user, ... := testSetupAUser()`

Reply:
if this method takes the testing object, you can do expectations in here and have a nice test failure if there's a problem. so like `testSetup(t *testing.T)`